_data/incubator.yml

```diff
@@ -7,9 +7,9 @@
 - title: "delta"
   description: "Type-safe, generic diffing of data"
   github: "https://github.com/stacycurl/delta"
-- title: "eff-cats"
-  description: "Extensible effects for Cats"
-  github: "https://github.com/etorreborre/eff-cats"
+- title: "eff"
+  description: "Extensible effects for Scala"
+  github: "https://github.com/etorreborre/eff"
```

Comment:
Shouldn't that be https://github.com/atnos-org/eff? https://github.com/etorreborre/eff redirects to https://github.com/atnos-org/eff-scalaz which is marked as deprecated.